apps/ui/src/components/user/info.tsx

```diff
@@ -45,14 +46,26 @@ export const Info = ({
     });
   };
 
+  useEffect(() => {
+    if (isMenuOpen) {
+      document.body.style.overflow = 'hidden';
+    } else {
+      document.body.style.overflow = 'auto';
+    }
+  }, [isMenuOpen]);
+
   // biome-ignore lint/correctness/useExhaustiveDependencies: <explanation>
   const icon = useMemo(() => {
     const path = localStorage.getItem(`@BAKO-ID/AVATAR/${account}`);
     return <Avatar borderRadius="lg" src={path!} />;
   }, []);
 
   return (
-    <Menu strategy="fixed">
+    <Menu
+      strategy="fixed"
+      onOpen={() => setIsMenuOpen(true)}
+      onClose={() => setIsMenuOpen(false)}
+    >
```

Comment:
you can do it this way to avoid relying on useState and useEffect:

```suggestion
const onOpenMenu = () => document.body.style.overflow = 'hidden';
const onCloseMenu = () =>  document.body.style.overflow = 'auto';

return (
    <Menu
      strategy="fixed"
      onOpen={onOpenMenu}
      onClose={onOpenMenu}
    >
```